tests/test_config.py

```diff
@@ -0,0 +1,97 @@
+import shlex
+import pytest
+import mock
+
+
+from jrnl.args import parse_args
+from jrnl.jrnl import run
+from jrnl import install
+from jrnl import jrnl
+
+
+@pytest.fixture()
+def minimal_config():
+    cfg = {
+        "colors": {"body": "red", "date": "green"},
+        "default": "/tmp/journal.jrnl",
+        "editor": "vim",
+        "encrypt": False,
+        "journals": {"default": "/tmp/journals/journal.jrnl"},
+    }
+    yield cfg
+
+
+@pytest.fixture()
+def expected_override(minimal_config):
+    exp_out_cfg = minimal_config.copy()
+    exp_out_cfg["editor"] = "nano"
+    exp_out_cfg["journal"] = "/tmp/journals/journal.jrnl"
+    yield exp_out_cfg
+
+
+@mock.patch("sys.stdin.isatty")
+@mock.patch.object(install, "load_or_install_jrnl")
+@mock.patch("subprocess.call")
+def test_override_configured_editor(
```

Comment:
This looks more like a behavorial test than a unit test - I believe it's already covered in the feature file. Ideally, the pytest unit test should be much smaller and testing individual functions or "units" of code, rather than large chunks of the program.